content/doc/pipeline/tour/hello-world.adoc

```diff
@@ -28,11 +28,13 @@ Handbook.
 
 To get started quickly with Pipeline:
 
+. Install the link:https://plugins.jenkins.io/docker-workflow/[Docker Pipeline] plugin through the *Manage Jenkins > Manage Plugins* page.
+. After installing the plugin, restart Jenkins so that plugin is ready to use.
 . Copy one of the <<examples, examples below>> into your repository and name it `Jenkinsfile`
 . Click the *New Item* menu within Jenkins
 image:pipeline/classic-ui-left-column.png[alt="Classic UI left column",width=40%]
-. Provide a name for your new item (e.g. *My-Pipeline*) and select *Multibranch Pipeline*
-. Click the *Add Source* button, choose the type of repository you want to use and fill in the details.
+. Provide a name for your new item (e.g. *My-Pipeline*) and select *Pipeline*. Click *Ok* to proceed further.
+. In the Pipeline section, copy and paste the simple Pipeline example with various languages from the *Quick Start Examples" section below.
```

Comment:
This change alters the purpose of the tutorial significantly.  It switches the tutorial from showing the user how to store their Pipeline definition in a git repository to instead store their Pipeline inside Jenkins.  I think that's a larger change than we want to make to the tutorial.
```suggestion
. Provide a name for your new item (e.g. *My-Pipeline*) and select *Multibranch Pipeline*
. Click the *Add Source* button, choose the type of repository you want to use and fill in the details.
```